data/generated.go

```diff
@@ -188,7 +188,7 @@ func isSourceMap(path, _ string, content []byte) bool {
 		return true
 	}
 
-	firstLine := getLines(content, 1)[0]
+	firstLine := getFirstLine(content)
 	for _, r := range sourceMapRegexps {
 		if r.Match(firstLine) {
 			return true
```

Comment:
it going to iterate even when the line is nil?

Reply:
Added an exit point for when it's empty